jams/eval.py

```diff
@@ -32,6 +32,9 @@ def validate_annotation(ann, namespace):
     '''Validate that the annotation has the correct namespace,
```

Comment:
It's not (really) a public-facing function, so I don't think so.

But, like I said, `coerce_and_validate` might be a more accurate description.